infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
+	Id            string   `json:"id"`
+	Title         string   `json:"title"`
+	Severity      string   `json:"severity"`
+	PublicId      string   `json:"publicId"`
+	Documentation string   `json:"documentation"`
+	LineNumber    int      `json:"lineNumber"`
+	Issue         string   `json:"issue"`
+	Impact        string   `json:"impact"`
+	Resolve       string   `json:"resolve:omitempty"`
+	References    []string `json:"references:omitempty"`
+	Path          []string `json:"path:omitempty"`
```

Comment:
What does this represent? Is it different to `FilePath` from `ScanIssue`?

Reply:
It looks the same, it should be the path to the issue in the affected file